boards/native/native_sim/native_sim.yaml

```diff
@@ -11,6 +11,7 @@ toolchain:
 supported:
   - can
   - counter
+  - display
```

Comment:
Maybe also add it to the native_posix yaml (It is deprecated, but the build of the driver is quite different in that case, so better to still built these on it)
Otherwise this looks fine to me.

Reply:
Done